src/data/data.tsx

```
@@ -12,14 +12,14 @@ import InstagramIcon from '../components/Icon/InstagramIcon';
 import LinkedInIcon from '../components/Icon/LinkedInIcon';
 import StackOverflowIcon from '../components/Icon/StackOverflowIcon';
 import profilepic from '../images/enio2024.png';
-import heroImage from '../images/istcok_watch_white.webp';
+import heroImage from '../images/white-20-to-strong-gray.webp';
```

ESLint

[error] '/home/jailuser/git/src/images/white-20-to-strong-gray.webp' imported multiple times.

(import/no-duplicates)